index.html

```diff
@@ -4,9 +4,29 @@
     <head>
         <meta charset="UTF-8" />
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <!--<meta name="description" content="Arthur Guzman | Portfolio, Front-End
+            Developer from Peru" />
+        <meta name="keywords" content="Portfolio,Front-End,Front-End
+            Developer,Personal Portfolio, HTML, CSS, Javascript, ArthurGC" />
+        <meta name="author" content="Arthur Guzman" />-->
```

Comment:
[OPTIONAL]
- [ ] Do not leave commented code in your codebase. They will make your project look messy